package.json

```diff
@@ -49,6 +49,7 @@
     "mem": "6.1.1",
     "nanoid": "3.1.22",
     "polished": "3.7.1",
+    "react-hook-form": "7.0.7",
```

Comment:
Should this be on latest RHF version ? https://github.com/react-hook-form/react-hook-form/releases/tag/v7.3.6